src/native-common/View.tsx

```diff
@@ -392,11 +398,18 @@ export class View extends ViewBase<Types.ViewProps, Types.Stateless> {
     }
 
     render() {
-        let PotentiallyAnimatedView = this._isButton(this.props) ? RN.Animated.View : RN.View;
+        let ViewToRender = RN.View;
+
+        if (this._isButton(this.props)) {
+            ViewToRender = RN.Animated.View;
+        } else if (this.props.useSafeInsets) {
+            ViewToRender = RN.SafeAreaView;
```

Comment:
Is SafeAreaView an Animated.View?

Reply:
I guess it doesn't matter since this was previously RN.View, but this means that buttons won't honor safe area insets?

Reply:
"useSafeInsets" is defined in ViewProps, not in ViewPropsShared, so it's not available to RX.Animated.View.